lib/dai-plugin-mcd/test/ManagedCdp.spec.js

```diff
@@ -128,6 +173,15 @@ describe.each([
       myDai: startingDaiBalance.plus(1),
       myGem: startingGemBalance.minus(1)
     });
+    const price = await cdp._cdpType.getPrice();
```

Comment:
seeing you use `_cdpType` like this, it seems reasonable to me that you might want to do this in ordinary code as well, in which case we might as well rename it to `type`